crypto/keys/secp256k1/internal/secp256k1/scalar_mult_cgo.go

```diff
@@ -21,7 +21,7 @@ extern int secp256k1_ext_scalar_mul(const secp256k1_context* ctx, const unsigned
 */
 import "C"
 
-func (BitCurve *BitCurve) ScalarMult(Bx, By *big.Int, scalar []byte) (*big.Int, *big.Int) {
```

Comment:
same thing here. You need to run tests with the build flag for these files to make it work. Otherwise we don't catch it

Reply:
ohhhhh, hey thank you on both accounts

Reply:
this should be resolved now, too.  I'll diff that whole folder to double check other files?